account_banking_pain_base/models/res_partner_bank.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Brain-Tec Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartnerBank(models.Model):
+    _inherit = 'res.partner.bank'
+
+    account_holder = fields.Char(default=' ')
+
+    @api.onchange('partner_id')
+    def _onchange_account_holder(self):
```

Comment:
This should be named as `_onchange_partner_id_ account_banking_pain_base`

Reply:
Why should it be this name? isn't it too long?

Reply:
Guidelines:

* Onchange name should be contain the field name of the field you are hooking it.
* It shouldn't be the general one (`_onchange_parner_id`), as other module can declare the same and choke both (only one will be called, which one? you don't know). The only way to assure your onchange name is unique is adding the module name to the usual method name.
* This is only needed when you are not the module that declares the field (which is the case). If you are declaring the field, no brother module can declare an onchange on your field without depending on your module.

Reply:
Thanks @pedrobaeza, I will change the name right away.